src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationIfElseWithNoCurly.java

```diff
@@ -0,0 +1,53 @@
+/* Config:                                                                    //indent:0 exp:0
+ *                                                                            //indent:1 exp:1
+ * arrayInitIndent = 4                                                        //indent:1 exp:1
+ * basicOffset = 4                                                            //indent:1 exp:1
+ * braceAdjustment = 0                                                        //indent:1 exp:1
+ * caseIndent = 4                                                             //indent:1 exp:1
+ * forceStrictCondition = false                                               //indent:1 exp:1
+ * lineWrappingIndentation = 4                                                //indent:1 exp:1
+ * tabWidth = 4                                                               //indent:1 exp:1
+ * throwsIndent = 4                                                           //indent:1 exp:1
+ *                                                                            //indent:1 exp:1
+ */                                                                           //indent:1 exp:1
+
+package com.puppycrawl.tools.checkstyle.checks.indentation.indentation; //indent:0 exp:0
+
+public class InputIndentationIfElseWithNoCurly { //indent:0 exp:0
```

Comment:
please copy this file few times and make "same" cases for `for/while/do-while/switch/case`

Reply:
Done

Reply:
made test cases for `for while and do-while`. switch is already there and had no issues like this with them.
They are already covered in there test cases, like: [InputIndentationInvalidSwitchIndent.java](https://github.com/checkstyle/checkstyle/blob/master/src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationInvalidSwitchIndent.java#L30-L46 )